src/content/docs/tutorial-improve-site-performance/guide-to-monitoring-core-web-vitals.mdx

```diff
@@ -76,63 +69,95 @@ Each web vital has the following thresholds to help you track your app's perform
   <tbody>
     <tr>
       <td>
-        <DoNotTranslate>**Largest contentful paint**</DoNotTranslate>
+        <DoNotTranslate>**Largest Contentful Paint**</DoNotTranslate>
       </td>
 
       <td>
-        &lt;=2.5s
+        2.5 seconds or less
       </td>
 
       <td>
-        &lt;=4s
+        Between 2.5 seconds and 5 seconds
       </td>
 
       <td>
-        &gt;4s
+        Longer than 4 seconds
       </td>
     </tr>
 
     <tr>
       <td>
-        <DoNotTranslate>**First input delay**</DoNotTranslate>
+        <DoNotTranslate>** Interaction to Next Paint** </DoNotTranslate>
```

Comment:
We sentance case this in the rest of the PR, might be a reason it's title case here but wanted to flag it

Reply:
Oh good catch, thank you!